### PR TITLE
Update MRVN to be up to date* with Net-Custom's

### PR DESCRIPTION
**Pull request by Baklojan**
I've updated everything manually so it shouldn't conflict
add few new stuff\brings Qol, fixes fixme's
-prop textures seem to fail to load

*there are few commits left from customs 
but they either build file changes/cmake changes/ quake3 map compiler changes which aren't even used for remap or fgd code changes which i'm scared to change

### CODE REVIEW

**Comment by Baklojan** [comment]
also shader editor code commits weren't implemented

**Comment by Baklojan** [comment]
https://github.com/Garux/netradiant-custom/commit/024204ab24feb27672f8f4c7a486256629c8268e
this would be good to add too
wayland section in README wouldn't be needed then

**Comment by snake-biscuits** [comment]
Good call, seems to just automate what we're asking users to do
Can't really think of any way that might break

File: contrib/bobtoolz/DPatch.cpp
Comment (snake-biscuits) [inline contrib/bobtoolz/DPatch.cpp:1]: changed the quote style, and missed the quotes on `")\t"`
why?

Comment (Baklojan) [reply contrib/bobtoolz/DPatch.cpp:1]: https://github.com/Garux/netradiant-custom/commit/df02774ff5a4276c990a87f6511eb9b96a9b11f7
There are plenty of missed quotes

**Pull request merged**